stern/tail.go

```diff
@@ -218,6 +218,10 @@ func (t *Tail) Print(msg string) {
 		return
 	}
 
+	if withHighlight {
+		fmt.Fprint(t.out, t.Options.HighlightMatchedString(buf.String()))
+		return
+	}
 	fmt.Fprint(t.out, buf.String())
```

Comment:
```suggestion
	fmt.Fprint(t.out, t.Options.HighlightMatchedString(buf.String()))
```

Some error messages may be highlighted, but I think the above change is fine.

Reply:
It's rare case, imho, but if it required, I can add feature to exclude `highlight` from errors